homeassistant/components/bond/config_flow.py

```diff
@@ -54,7 +56,8 @@ async def async_step_user(self, user_input=None):
                 _LOGGER.exception("Unexpected exception")
                 errors["base"] = "unknown"
             else:
-                return self.async_create_entry(title=info["title"], data=user_input)
+                await self.async_set_unique_id(bond_id)
+                return self.async_create_entry(title=bond_id, data=user_input)
```

Comment:
How are we guarding from creating more than one config entry with the same bond id? We normally call `self._abort_if_unique_id_configured` after setting unique id.

Reply:
I added it here: https://github.com/home-assistant/core/pull/38416